Bender.yml

```diff
@@ -7,44 +7,87 @@ package:
 
 # WT_DCACHE
 
-export_include_dirs:
-  - vendor/pulp-platform/common_cells/include/
-  - corev_apu/axi/include/
+dependencies:
+  register_interface: { git: "git@github.com:pulp-platform/register_interface.git", version: 0.3.3 }
+  axi: { git: "git@github.com:pulp-platform/axi.git", version: 0.37.0 }
+  axi_riscv_atomics: { git: "git@github.com:pulp-platform/axi_riscv_atomics.git", version: 0.5.0 }
+  tech_cells_generic: { git: "git@github.com:pulp-platform/tech_cells_generic.git", version: 0.2.9 }
+  riscv-dbg: { git: "git@github.com:pulp-platform/riscv-dbg", version: 0.7.0 }
+  common_cells: { git: "git@github.com:pulp-platform/common_cells", version: 1.25.0 }
```

Comment:
I just briefly discussed this with @micprog. Our convention is to use `https://<path>.git` as a format for specifying the dependencies